cabal.project.user

```diff
@@ -3,12 +3,12 @@ packages: streamly-process.cabal
 source-repository-package
    type: git
    location: https://github.com/composewell/streamly.git
-   tag: master
+   tag: 24926eb041da4bc25c55460e0ecbf18fcb33e990
```

Comment:
This is an old commit of streamly. We should not use source-repository stanza, instead allow cabal to use it from hackage.